packages/at_auth/lib/src/at_auth_impl.dart

```diff
@@ -74,9 +74,9 @@ class AtAuthImpl implements AtAuth {
     }
     atLookUp ??= AtLookupImpl(
         atAuthRequest.atSign, atAuthRequest.rootDomain, atAuthRequest.rootPort);
-    var atChops = _createAtChops(atAuthKeys);
-    this.atChops = atChops;
-    atLookUp!.atChops = atChops;
+    atLookUp?.signingAlgoType = atAuthRequest.signingAlgoType;
+    atLookUp?.hashingAlgoType = atAuthRequest.hashingAlgoType;
+    atLookUp!.atChops = atChops = _createAtChops(atAuthKeys);
```

Comment:
is there a functional test which verifies this change?

Reply:
Not yet @Muralidharan. Let me write one up.

Reply:
A functional test has been added but is skipped for now, as it has to set a hashing algo other than SHA256. And at_server does not recognize any hashin algo other than SHA256.